### ISSUE TITLE
[BUG] Trojan:Script/Oneeva.A!ml

### ISSUE THREAD

**Issue by Wyrade** [ColorlibHQ/AdminLTE#3135]
**Describe the bug**
After downloading the master branch through the "Code->Download ZIP" button, windows defender indicated the zip as a "Trojan:Script/Oneeva.A!ml". I assume this is not normal.

**To Reproduce**
See above.

**Expected behavior**
Not being recognized as a trojan.

**Environment (please complete the following information):**
- AdminLTE Version: v3.0.5
- Operating System:  Windows 10 Enterprise 20H2
- Browser (Version): Mozilla Firefox 82.0.3

**Additional context**
N/A

**Comment by REJack** [comment]
I can't replicate this, even [VirusTotal](https://www.virustotal.com/gui/file/e1a9e958cb362a6280dc117ba0392f03a500c7f0b0b67fb073a4637762b19237/detection) didn't found a Trojan.

![grafik](https://user-images.githubusercontent.com/1554582/99742101-11cc4d80-2ad3-11eb-8927-a4342dc91c16.png)

**Comment by Wyrade** [comment]
So originally I checked this on my laptop, which threw the virus alert for the zip.
I checked now on my home PC, a Win10 Pro 1909, which does not detect it as a virus.
I checked it on my home PC in Oracle VM with a Win10 Enterprise 20H2, which does detect it as a virus.
After extraction on the VM it does not detect any file as a virus, only the unextracted zip itself.
The hash of the zip on the VM and my home PC seem to be the same.

I tried downloading the codeql branch instead of the master to see what happens, that zip doesn't get detected as a virus on the VM.
I also tried downloading something completely unrelated's master branch from github - that didn't get detected as a virus either.
If I zip the extracted files myself with a 7zip it doesn't get detected as a virus either.

Idk what's going on, but I can replicate my issue.
